package.json

```diff
@@ -13,6 +13,7 @@
     "jss-rtl": "^0.3.0",
     "node-sass": "^4.13.1",
     "react": "^16.13.1",
+    "react-commons": "file:react-commons-0.1.0.tgz",
```

Comment:
Excellent 
It would better to put it in a folder not in project root.

Reply:
in src/asset ?